### DISCUSSION TITLE
`webpack-hmr` never loads when `assetPrefix` is non-empty string

### DISCUSSION THREAD

**Discussion by thany** [vercel/next.js#63817] [Help]
### Summary

The bug that I filed was closed:

https://github.com/vercel/next.js/issues/63816

Bot says repro link is not correct. Not true, the repro link is correct.

### Additional information

Please reopen the bug, or maybe answer here. Either is fine with me.

**Comment by thany** [comment]
Created a new bug. I would appreciate it if it didn't get closed again. Especially because it's impossible to copy an existing bug into the template. It took me 2 minutes too long just because of that silly bot. It's not much time, but it's a lot of frustration. I'm already dealing with a weird error. Keep that in mind when willy-nilly closing someone's bug.

I could've just edited the bug, you know...

**Comment by Netail** [comment]
Duplicate of https://github.com/vercel/next.js/issues/63623
Should be fixed with https://github.com/vercel/next.js/pull/63627
